### PR TITLE
[STACK move] Move applicable functions from sum_doses experimental app to library

### PR DESCRIPTION
**Pull request by Matthew-Jennings**

### CODE REVIEW

File: lib/pymedphys/_dicom/coords.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Copyright (C) 2019 Matthew Jennings
+# Copyright (C) 2019-21 Matthew Jennings
```
Comment (SimonBiggs) [inline lib/pymedphys/_dicom/coords.py:1]:
```suggestion
# Copyright (C) 2019, 2021 Matthew Jennings
```

Or `# Copyright (C) 2019-2021 Matthew Jennings`

**Review by SimonBiggs — APPROVED** [review]
Awesome :) Thanks @Matthew-Jennings, if these are all just moves alone then they're good for merge :slightly_smiling_face:

File: lib/pymedphys/_dicom/coords.py (hunk 1 above)
Comment (Matthew-Jennings) [reply lib/pymedphys/_dicom/coords.py:1]: Fair enough!

**Comment by SimonBiggs** [comment]
> Awesome :) Thanks @Matthew-Jennings, if these are all just moves alone then they're good for merge

...actually... oops. We should probably wait until the other PR is approved before merging... to keep to standard library standards :slightly_smiling_face:.

**Comment by Matthew-Jennings** [comment]
> > Awesome :) Thanks @Matthew-Jennings, if these are all just moves alone then they're good for merge
> 
> ...actually... oops. We should probably wait until the other PR is approved before merging... to keep to standard library standards 🙂.

No worries. No rush!

**Comment by Matthew-Jennings** [comment]
Test failures will be fixed by #1405

**Pull request merged**